src/Sulu/Bundle/MediaBundle/Resources/js/containers/MultiMediaDropzone/tests/MultiMediaDropzone.test.js

```diff
@@ -20,9 +20,15 @@ jest.mock('sulu-admin-bundle/utils', () => ({
 }));
 
 jest.mock('../../../stores/MediaUploadStore', () => jest.fn(function() {
-    this.create = jest.fn().mockReturnValue(Promise.resolve({
-        id: 123,
-    }));
+    this.create = jest.fn((_, file) => {
```

Comment:
Can't we put that into the test we do this for? This way it is hard to tell which tests need that, and it gets messier the bigger this file gets.

Reply:
I am not sure how to do that unfortunately 😕 
The store is created inside of the `handleDrop` method of the component and the `MediaUploadStore.create` method is called in the next statement. Is there a way to mock this inside of the testcase?

https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/MediaBundle/Resources/js/containers/MultiMediaDropzone/MultiMediaDropzone.js#L75-L76

Reply:
The `MediaUploadStore` has already been imported above, so you should be able to change the return value of the mocked function in the test itself somehow like this (not tested):

```javascript
MediaUploadStore.mock.instances[index].create.mockReturnValue(...);
```

Reply:
unfortunately calling `console.log(MediaUploadStore.mock.instances);` before the `onDrop` call logs an empty array (`[]`) 🙈

Reply:
Then the only other way I see around this, is to replace the entire constructor mock... But yeah, that might be a bit cumbersome too... Let's merge it like this, and hope it never makes any problems :see_no_evil: